### PR TITLE
add import check to travis

### PR DESCRIPTION
**Pull request by laizy**
goimports -format-only -w .

### CODE REVIEW

File: core/store/ledgerstore/block_store.go [hunk 1]
```diff
@@ -23,12 +23,13 @@ import (
 	"encoding/binary"
 	"fmt"
```
Comment (Honglei-Cong) [inline core/store/ledgerstore/block_store.go]: blank line

File: core/store/ledgerstore/ledger_store.go [hunk 2]
```diff
@@ -33,11 +30,15 @@ import (
 	"sync"
 	"time"
 
+	"github.com/ontio/ontology/merkle"
+	"github.com/ontio/ontology/smartcontract/service/native/utils"
+	types2 "github.com/ontio/ontology/vm/neovm/types"
+
```
Comment (Honglei-Cong) [inline core/store/ledgerstore/ledger_store.go]:
```suggestion
```

File: http/jsonrpc/rpc_server.go [hunk 3]
```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
```
Comment (Honglei-Cong) [inline http/jsonrpc/rpc_server.go:26]:
```suggestion
```

File: http/localrpc/local_server.go [hunk 4]
```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
```
Comment (Honglei-Cong) [inline http/localrpc/local_server.go:26]:
```suggestion
```

File: smartcontract/service/native/cross_chain/cross_chain_manager/cross_chain.go [hunk 5]
```diff
@@ -21,10 +21,12 @@ package cross_chain_manager
 import (
 	"encoding/hex"
 	"fmt"
-	"github.com/ontio/ontology/smartcontract/service/native/cross_chain/header_sync"
 	"math/big"
 
+	"github.com/ontio/ontology/smartcontract/service/native/cross_chain/header_sync"
+
 	"bytes"
```
Comment (Honglei-Cong) [inline smartcontract/service/native/cross_chain/cross_chain_manager/cross_chain.go]: fix here?

File: smartcontract/service/native/cross_chain/cross_chain_manager/utils.go [hunk 6]
```diff
@@ -21,6 +21,7 @@ package cross_chain_manager
 import (
 	"encoding/hex"
 	"fmt"
+
 	"github.com/ontio/ontology/common"
```
Comment (Honglei-Cong) [inline smartcontract/service/native/cross_chain/cross_chain_manager/utils.go]:
```suggestion
```

File: smartcontract/service/native/cross_chain/lock_proxy/states_test.go [hunk 7]
```diff
@@ -22,11 +22,12 @@ import (
 	"testing"
```
Comment (Honglei-Cong) [inline smartcontract/service/native/cross_chain/lock_proxy/states_test.go]:
```suggestion
```

File: smartcontract/service/native/native_service.go [hunk 8]
```diff
@@ -20,6 +20,7 @@ package native
 
 import (
 	"fmt"
+
 	"github.com/ontio/ontology/merkle"
```
Comment (Honglei-Cong) [inline smartcontract/service/native/native_service.go]:
```suggestion
```

File: smartcontract/service/native/ong/ong.go [hunk 9]
```diff
@@ -22,6 +22,7 @@ import (
 	"math/big"
```
Comment (Honglei-Cong) [inline smartcontract/service/native/ong/ong.go]:
```suggestion
```

File: smartcontract/service/native/testsuite/ont_suite.go [hunk 10]
```diff
@@ -20,10 +20,11 @@ package testsuite
 import (
 	"crypto/rand"
 	"encoding/hex"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+
```
Comment (Honglei-Cong) [inline smartcontract/service/native/testsuite/ont_suite.go]:
```suggestion
```

File: smartcontract/service/neovm/wasmvm.go [hunk 11]
```diff
@@ -19,9 +19,10 @@ package neovm
 
 import (
 	"fmt"
-	"github.com/ontio/ontology/core/utils"
 	"reflect"
 
+	"github.com/ontio/ontology/core/utils"
+
```
Comment (Honglei-Cong) [inline smartcontract/service/neovm/wasmvm.go]:
```suggestion
```

File: smartcontract/service/util/util.go [hunk 12]
```diff
@@ -21,6 +21,7 @@ import (
 	"bytes"
```
Comment (Honglei-Cong) [inline smartcontract/service/util/util.go]:
```suggestion
```

File: smartcontract/service/wasmvm/runtime.go [hunk 13]
```diff
@@ -23,6 +23,8 @@ import (
 	"fmt"
 	"reflect"
```
Comment (Honglei-Cong) [inline smartcontract/service/wasmvm/runtime.go]:
```suggestion
```

File: wasmtest/common/common.go [hunk 14]
```diff
@@ -21,6 +21,7 @@ import (
 	"bytes"
```
Comment (Honglei-Cong) [inline wasmtest/common/common.go]:
```suggestion
```

File: wasmtest/common/common.go [hunk 15]
```diff
@@ -29,6 +30,7 @@ import (
 
 	"github.com/ontio/ontology/common"
 	"github.com/ontio/ontology/core/types"
+
```
Comment (Honglei-Cong) [inline wasmtest/common/common.go]:
```suggestion
```

File: wasmtest/common/common.go [hunk 16]
```diff
@@ -29,6 +30,7 @@ import (
...
 	// neovms "github.com/ontio/ontology/smartcontract/service/neovm"
```
Comment (Honglei-Cong) [inline wasmtest/common/common.go]:
```suggestion
```

File: core/store/ledgerstore/block_store.go (hunk 1 above)
Comment (laizy) [inline core/store/ledgerstore/block_store.go]:
```suggestion
```

File: smartcontract/service/native/cross_chain/cross_chain_manager/cross_chain.go [hunk 17]
```diff
@@ -21,10 +21,12 @@ package cross_chain_manager
...
+
 	"github.com/ontio/ontology/common"
```
Comment (laizy) [inline smartcontract/service/native/cross_chain/cross_chain_manager/cross_chain.go]:
```suggestion
	"math/big"
	"bytes"

	"github.com/ontio/ontology/smartcontract/service/native/cross_chain/header_sync"
	"github.com/ontio/ontology/common"
```

File: common/address.go [hunk 18]
```diff
@@ -24,9 +24,10 @@ import (
 	"fmt"
 	"math/big"
```
Comment (Honglei-Cong) [inline common/address.go]:
```suggestion
```

File: common/serialization/serialize_test.go [hunk 19]
```diff
@@ -26,6 +26,7 @@ import (
 	"testing"
```
Comment (Honglei-Cong) [inline common/serialization/serialize_test.go:28]:
```suggestion
```

File: common/zero_copy_sink_test.go [hunk 20]
```diff
@@ -21,6 +21,7 @@ import (
 	"testing"
```
Comment (Honglei-Cong) [inline common/zero_copy_sink_test.go]:
```suggestion
```

File: core/payload/bookkeeper_test.go [hunk 21]
```diff
@@ -22,6 +22,7 @@ import (
 	"testing"
```
Comment (Honglei-Cong) [inline core/payload/bookkeeper_test.go]:
```suggestion
```

File: vm/neovm/types/int_value.go [hunk 22]
```diff
@@ -21,11 +21,12 @@ package types
 import (
 	"math/big"
```
Comment (Honglei-Cong) [inline vm/neovm/types/int_value.go]:
```suggestion
```

**Review by Honglei-Cong — APPROVED** [review]

**Pull request merged**